tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/globals.py

```diff
@@ -134,44 +137,29 @@
     "tissue_ontology_term_id",
     "tissue_general",
     "tissue_general_ontology_term_id",
+    "tissue_type",
+]
+_NonRepeatitiveStringObs = [
```

Comment:
Typo: this should be `_NonRepetitiveStringObs` and the other variable `_RepetitiveStringLabelObs` (the spelling isn't consistent).

Reply:
This code all goes away in #896, which is stacked on top of this PR.  So I'd propose to ignore this issue in favor of the changes in that PR.  LMK if you are not OK with that.

Reply:
Totally OK. 🚀